ci/publish_packages.sh

```diff
@@ -2,18 +2,26 @@
 
 set -e -u -o pipefail
 
+if [[ -n "${DEBUG-}" ]]; then
+  set -x
+fi
+
 cd "$(dirname "$0")/../"
 
 PKG_ROOT="workers/unity/Packages"
-REGISTRY="https://npm.improbable.io/gdk-for-unity/"
+
+./init.sh
 
 # Re-publish all packages
-pushd $PKG_ROOT
-    for dir in */;
+pushd "${PKG_ROOT}" > /dev/null
+    for package_dir in */;
     do
-        echo $dir
-        pushd $dir
-            npm publish --registry="${REGISTRY}"
-        popd
+        echo "${package_dir}"
+        pushd "${package_dir}" > /dev/null
+            # Created package is the last line of output.
+            PACKAGE_NAME=$(npm pack | tail -n 1)
+            cloudsmith push npm spatialos/gdk-for-unity "${PACKAGE_NAME}" --republish
```

Comment:
Is `--republish` something cloudsmith-specific?

Reply:
Yes - it allows packages to overwrite existing versions. Otherwise if we find a regression we must manually pull them all.